infrastructure/main.tf

```diff
@@ -54,9 +54,6 @@ locals {
   s2s_vault_url     = "https://s2s-${local.local_env}.vault.azure.net/"
 
   core_app_settings = {
-    LOGBACK_REQUIRE_ALERT_LEVEL = "false"
```

Comment:
is this not required anymore ?

Reply:
completely annihilated since [4.0.0](https://github.com/hmcts/java-logging/releases/tag/4.0.0)